nemo/collections/llm/inference/base.py

```diff
@@ -60,6 +86,15 @@ def _setup_trainer_and_restore_model(path: Path, trainer: nl.Trainer, model: pl.
     trainer.strategy.trainer = trainer
     trainer.strategy.selective_restore()
 
+    lora: Union[io.TrainerContext, LoRA] = io.load_context(ckpt_to_context_subdir(path), "model.model_transform")
```

Comment:
As of this morning I tried train peft, only the -last checkpoint has context subdir?

Reply:
you're right, but yeah we have to load the context here.